packages/mdc-drawer/dismissible/mdc-drawer-dismissible.scss

```diff
@@ -17,62 +17,30 @@
 @import "@material/animation/functions";
 @import "@material/rtl/mixins";
 @import "../variables";
+@import "../mixins";
 
 .mdc-drawer--dismissible {
-  width: 0;
-  min-width: 0;
-  transform: translateX(-100%);
-  border-right-width: 0;
+  @include mdc-rtl-reflexive-position(left, 0);
 
-  @include mdc-rtl {
-    transform: translateX(100%);
-    border-left-width: 0;
-  }
-}
+  display: none;
+  position: absolute;
+  top: 0;
+  transition-property: transform;
```

Comment:
transition-property and transition-timing-function are already applied to `.mdc-drawer` in the standard styles so they're redundant here.

On a separate topic (for a separate PR), maybe it'd be a better idea to put any common drawer styles directly within `mdc-drawer.scss` (instead of under `standard`) to make it evident that it has an effect on all variants.